### ISSUE TITLE
Bug]: Fatal Error 

### ISSUE THREAD

**Issue by atomiccherry** [thewebforge/structeezy-docs#12]
### Contact Details

john@atomiccherry.com

### What happened?

A bug happened!

### Version

v1.2.0

### Version

v1.9.5

### What browsers are you seeing the problem on?

Brave

### Relevant log output

```shell
Fatal error: Uncaught Error: Call to undefined function get_plugins() in /Users/Lucy/Local Sites/bricks/app/public/app/content/plugins/structeezy/includes/frameworks.php:31 Stack trace: #0 /Users/Lucy/Local Sites/bricks/app/public/app/content/plugins/structeezy/includes/frameworks.php(46): structeezy_is_known_framework() #1 /Users/Lucy/Local Sites/bricks/app/public/app/content/plugins/structeezy/structeezy.php(77): require_once('/Users/Lucy/Loc...') #2 /Users/Lucy/Local Sites/bricks/app/public/app/wp-settings.php(473): include_once('/Users/Lucy/Loc...') #3 /Users/Lucy/Local Sites/bricks/app/public/app/wp-config.php(169): require_once('/Users/Lucy/Loc...') #4 /Users/Lucy/Local Sites/bricks/app/public/app/wp-load.php(50): require_once('/Users/Lucy/Loc...') #5 /Users/Lucy/Local Sites/bricks/app/public/app/wp-admin/admin.php(34): require_once('/Users/Lucy/Loc...') #6 /Users/Lucy/Local Sites/bricks/app/public/app/wp-admin/plugins.php(10): require_once('/Users/Lucy/Loc...') #7 {main} thrown in /Users/Lucy/Local Sites/bricks/app/public/app/content/plugins/structeezy/includes/frameworks.php on line 31
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cbontems** [comment]
Thanks for reporting. Let me check what happens here.

**Comment by cbontems** [comment]
Hello again @atomiccherry ,

I just sent you an email with a patched plugin.

I could not reproduce the error on my test environment, but from what I saw in your log, this should solve it.

Can you please let me know if it solves the problem for you?

Thanks!

**Comment by cbontems** [comment]
Fixed with release of v1.2.1
